k8s/aat/software-engineering/rpe-feature-toggle-api/rpe-feature-toggle-api.yaml

```
@@ -18,6 +18,7 @@ spec:
       applicationInsightsInstrumentKey: "fac0a65a-a152-4aff-952c-8aec63b83de3"
       image: hmcts.azurecr.io/hmcts/feature-toggle-api:latest
       ingressHost: rpe-feature-toggle-api.service.core-compute-aat.internal
+      releaseNameOverride: rpe-feature-toggle-api
```

you could probably set this to `nil`
```suggestion
      releaseNameOverride: nil
```

and let the default be used here